https://github.com/qooxdoo/qooxdoo/issues/10018 qx.ui.tree.VirtualTre…

ofilippov opened the pull request:
…e: method isShowTopLevelOpenCloseIcons always returns true and ignores property showTopLevelOpenCloseIcons

*Solution*

Method isShowTopLevelOpenCloseIcons of class qx.ui.tree.VirtualTree should return the same value as a method getShowTopLevelOpenCloseIcons

ofilippov committed:
https://github.com/qooxdoo/qooxdoo/issues/10018 qx.ui.tree.VirtualTree: method isShowTopLevelOpenCloseIcons always returns true and ignores property showTopLevelOpenCloseIcons

*Solution*

Method isShowTopLevelOpenCloseIcons of class qx.ui.tree.VirtualTree should return the same value as a method getShowTopLevelOpenCloseIcons

derrell commented on framework/source/class/qx/ui/tree/VirtualTree.js:
```diff
@@ -608,7 +608,7 @@ qx.Class.define("qx.ui.tree.VirtualTree",
     // Interface implementation
     isShowTopLevelOpenCloseIcons : function() {
-      return true;
+      return this.getShowTopLevelOpenCloseIcons();
```
I believe this whole function is unnecessary. I believe it is overriding the function of the same name which is automatically-generated for boolean properties. I believe we should probably remove this method entirely, if my recollection of the automatically-created methods is accurate...

ofilippov replied:
Yes, I agree. I've removed the whole method. Thanks!

ofilippov committed:
https://github.com/qooxdoo/qooxdoo/issues/10018 qx.ui.tree.VirtualTree: method isShowTopLevelOpenCloseIcons always returns true and ignores property showTopLevelOpenCloseIcons

*Solution*

We don't need isShowTopLevelOpenCloseIcons here, it is auto-generated from property
